### PR TITLE
feat: Added Prettier configuration.

### PR DESCRIPTION
**Pull request by Priyansh1825**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +71,19 @@
   },
   "devDependencies": {
     "@types/styled-components": "^5.1.26",
-    "react-router-dom": "^6.8.2"
+    "@typescript-eslint/eslint-plugin": "^5.55.0",
+    "@typescript-eslint/parser": "^5.55.0",
+    "eslint-config-prettier": "^8.7.0",
```
Comment (vaidehi27) [inline package.json]: Remove this plugin

File: package.json [hunk 2]
```diff
@@ -11,15 +26,25 @@
...
     "@types/react-dom": "^18.0.11",
     "bootstrap": "^5.2.3",
+    "lint-staged": "^13.2.0",
+    "prettier": "^2.8.4",
     "react": "^18.2.0",
     "react-bootstrap": "^2.7.2",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
     "styled-components": "^5.3.6",
+    "stylelint-config-recommended": "^11.0.0",
```
Comment (vaidehi27) [inline package.json]: Remove this plugin

File: package.json [hunk 3]
```diff
@@ -2,6 +2,21 @@
...
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint:fix && format:fix"
+    }
+  },
+  "lint-staged": {
+    "*.{ts,tsx,json,md}": [
+      "npm run format"
+    ]
+  },
```
Comment (vaidehi27) [inline package.json]: is this needed?

File: tsconfig.json [hunk 4]
```diff
@@ -22,5 +22,5 @@
   },
   "include": [
     "src"
-  ]
+, "stylelint.config.js"  ]
```
Comment (vaidehi27) [inline tsconfig.json]: Is this needed?

File: src/Pages/About/style.ts [hunk 5]
```diff
@@ -2,7 +2,7 @@ import styled from "styled-components";
 
 export const AboutUsText = styled.div`
   .container {
-    margin: 50px;
+    margin: "50px";
```
Comment (vaidehi27) [inline src/Pages/About/style.ts]:
```suggestion
    margin: 50px;
```

File: package.json (hunk 1 above)
Comment (Neel0851) [reply package.json]: okay

File: package.json (hunk 2 above)
Comment (Neel0851) [reply package.json]: okay

File: tsconfig.json (hunk 4 above)
Comment (Neel0851) [reply tsconfig.json]: no

File: package.json (hunk 3 above)
Comment (Priyansh1825) [reply package.json]: no

File: src/Pages/About/style.ts (hunk 5 above)
Comment (Priyansh1825) [reply src/Pages/About/style.ts]: okay

**Review by vaidehi27 — APPROVED** [review]

**Pull request merged**